aries-site/src/layouts/content/Example/Example.js

```diff
@@ -142,9 +142,8 @@ export const Example = ({
     else if (!fullscreen) {
       const containerWidth = mockBrowserRect.width;
       scaledTheme = screenContainer.scale
-        ? scaled(theme, screenContainer.scale)
+        ? scaled(undefined, screenContainer.scale)
```

Comment:
Is there anywhere else we're calling `scaled` with another theme? If not, should we just let `scaled()` use aries theme and remove the option for a theme parameter?

Reply:
We can. I will.

I was originally using this to test out the concept where we could repurpose for other use cases. We can always add back at a latter date.

Reply:
Done.